src/settings/tabenvironment.cpp

```diff
@@ -0,0 +1,90 @@
+#include "tabenvironment.h"
+
+#include <QtWidgets>
+
+TabEnvironment::TabEnvironment(QWidget *parent)
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed